### PR TITLE
Added more aerospike flags

### PR DESCRIPTION
**Pull request by ehankland**

### CODE REVIEW

File: perfkitbenchmarker/benchmarks/aerospike_benchmark.py [hunk 1]
```diff
@@ -40,6 +40,19 @@
 flags.DEFINE_enum('aerospike_storage_type', MEMORY, [MEMORY, DISK],
                   'The type of storage to use for Aerospike data. The type of '
                   'disk is controlled by the "scratch_disk_type" flag.')
+flags.DEFINE_integer('aerospike_min_client_threads', 8,
```
Comment (yuyantingzero) [inline perfkitbenchmarker/benchmarks/aerospike_benchmark.py]: Should these also be written into metadata?

Comment (ehankland) [inline perfkitbenchmarker/benchmarks/aerospike_benchmark.py]: The number of client threads is already written into metadata - but the read percent should be and I dont think that it is - I'll fix that.

Comment (ehankland) [inline perfkitbenchmarker/benchmarks/aerospike_benchmark.py]: Done

**Comment by yuyantingzero** [comment]
LGTM

**Pull request merged**